editors/code/src/commands/server_version.ts

```diff
@@ -1,20 +1,10 @@
-import * as vscode from 'vscode';
-import { ensureServerBinary } from '../installation/server';
+import * as vscode from "vscode";
+import { spawnSync } from "child_process";
 import { Ctx, Cmd } from '../ctx';
-import { spawnSync } from 'child_process';
 
 export function serverVersion(ctx: Ctx): Cmd {
     return async () => {
-        const binaryPath = await ensureServerBinary(ctx.config, ctx.state);
-
-        if (binaryPath == null) {
-            throw new Error(
-                "Rust Analyzer Language Server is not available. " +
-                "Please, ensure its [proper installation](https://rust-analyzer.github.io/manual.html#installation)."
-            );
-        }
-
-        const version = spawnSync(binaryPath, ["--version"], { encoding: "utf8" }).stdout;
+        const version = spawnSync(ctx.serverPath, ["--version"], { encoding: "utf8" }).stdout;
```

Comment:
Maybe we should still use `getServer()` thing here?

Reply:
This will require adding persistentState to ctx, which I’d rather avoid.